src/Microsoft.DotNet.Build.Tasks/PackageFiles/RunnerTemplate.Unix.txt

```diff
@@ -89,8 +89,6 @@ fi
 
 # ========================= BEGIN Test Execution =============================
 echo Running tests... Start time: $(date +"%T")
-echo Commands:
-[[TestRunCommandsEcho]]
```

Comment:
This is going to make logs lose the commands called... doesn't affect me because I'm fine finding the scripts themselves but I'm not a fan of this solution.